Packs/QRadar/Integrations/QRadar_v2/QRadar_v2.py

```diff
@@ -910,7 +910,7 @@ def fetch_raw_offenses(client: QRadarClient, offense_id, user_query):
     """
     # try to adjust start_offense_id to user_query start offense id
     try:
-        if isinstance(user_query, str) and "id>" in user_query:
```

Comment:
I suggest you to use the regex I have made in QRadar v3, see ID_QUERY_REGEX. WDYT?

Reply:
Great suggestion